scalabel/eval/mot.py

```diff
@@ -313,12 +364,11 @@ def evaluate_track(
         nproc: processes number for loading files
 
     Returns:
-        dict: CLEAR MOT metric scores
+        BoxTrackResult: rendered eval results.
     """
     logger.info("Tracking evaluation with CLEAR MOT metrics.")
```

Comment:
this logger should be argument to the function, s.t. the user can decide where / if to print